artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/tools/PrintData.java

```diff
@@ -124,6 +124,10 @@ public static void printData(File bindingsDirectory, File messagesDirectory, Fil
       printData(bindingsDirectory, messagesDirectory, pagingDirectory, System.out, secret, false);
    }
 
+   public static void printData(File bindingsDirectory, File messagesDirectory, File pagingDirectory, PrintStream out, boolean secret) throws Exception {
```

Comment:
I did not know this method was in use at wildly. sorry about that.. I though it was only used internally.